charts/hedgedoc/templates/deployment.yaml

```diff
@@ -45,6 +46,29 @@ spec:
           value: "5432"
         - name: CMD_DB_DIALECT
           value: postgres
+        {{- else if .Values.postgresqlExternal.enabled }}
+        - name: CMD_DB_USERNAME
+          valueFrom:
+            secretKeyRef:
+              key: database-user
+              name: postgres
+        - name: CMD_DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: database-password
+              name: postgres
+        - name: CMD_DB_DATABASE
+          valueFrom:
+            secretKeyRef:
+              key: database-name
+              name: postgres
+        - name: CMD_DB_HOST
+          value: {{ .Values.postgresqlExternal.externalDatabase }}
+        - name: CMD_DB_PORT
+          value: "5432"
+        - name: CMD_DB_DIALECT
+          value: postgres
```

Comment:
Isn't this and other parameters repeated for both options?

Reply:
Thanks 👍 . It's cleaned now